internal/datastore/spanner/stats.go

```diff
@@ -35,7 +35,7 @@ func (sd spannerDatastore) Statistics(ctx context.Context) (datastore.Stats, err
 		ctx,
 		tableNamespace,
 		spanner.AllKeys(),
-		[]string{colNamespaceConfig},
+		[]string{colNamespaceConfig, colNamespaceTS},
```

Comment:
stats probably not that important, but for large instances this query could have impact capacity used-wise

Reply:
how would it be that different from the previous one?

Reply:
if there is an index, it shouldn't, but it may lead to an additional disk access if the request is not being served from the index anymore